File: Day 1/Chidamber & Kemerer (CK) Metric Suite/RFC.java
class TaskManager {
    public void start() {
        System.out.println("Start Task");
        helper1();
        helper2();
        //_logger.log();      // external call
        //_db.save();         
        //_email.send();      
    }

    void helper1() { System.out.println("Helper 1 working"); }
    void helper2() { System.out.println("Helper 2 working"); }
}

//2 internal+1 external method call so RFC 3)
class CleanTaskManager {
    public void start() {
        System.out.println("Start Task");
        helper1();
        helper2();
       // _facade.handle();  
    }

    void helper1() { System.out.println("Helper 1 working"); }
    void helper2() { System.out.println("Helper 2 working"); }
}

class RFC {
    public static void main(String[] args) {
        new TaskManager().start();
        new CleanTaskManager().start();
    }
}

File: Day 1/Chidamber & Kemerer (CK) Metric Suite/RFC.ts
class TaskManagerTS {
    start() {
        console.log("Start Task");
        this.helper1();
        this.helper2();
        //logger.log();
       // db.save();
        //email.send();
    }

    helper1() { console.log("Helper 1 working"); }
    helper2() { console.log("Helper 2 working"); }
}

// Low RFC
class CleanTaskManagerTS {
    start() {
        console.log("Start Task");
        this.helper1();
        this.helper2();
        //facade.handle();
    }

    helper1() { 
        console.log("Helper 1 working"); 
    }
    helper2() { 
        console.log("Helper 2 working"); 
    }
}

new TaskManagerTS().start();
new CleanTaskManagerTS().start();